README.EN.md

```diff
@@ -0,0 +1,39 @@
+[**Wersja polska**][ext0]
+# OpenCart Payment Extension for Paynow
+
+This plugin supports OpenCart 3.0 and higher.
+
+To access Paynow API please contact our support at kontakt@paynow.pl.
```

Comment:
To stwierdzenie chyb nie jest już potrzebne.

Reply:
narazie wszędzie mamy takie stwierdzenie, usunąć to?